event hubs migration guide.md

```diff
@@ -0,0 +1,86 @@
+# Migration Guide (EventHubs 2 to 5)
+
+## Class changes
+
+`EventHubClient` has been split into two clients - `EventHubProducerClient` (for sending
```

Comment:
We need to also talk about how the EPH class is not needed anymore and is replaced by EventHubConsumerClient

Reply:
Added a line (haven't done the sample yet)

Reply:
Looks like this is a duplicate comment to below. Resolving this one.